Dockerfile

```diff
@@ -29,6 +29,9 @@ COPY --from=build-dependencies /src/node_modules /src/node_modules
 COPY . ./
 RUN npm run build
 
+# Copy start shell script
+COPY ../start.sh ./
```

Comment:
Done as mentioned within the platform docs guidelines "Use SealedSecret as environment variable".